plugins/mntre/fu-mntre-device.c

```diff
@@ -0,0 +1,181 @@
+/*
+ * Copyright 2024 Chris hofstaedtler <Ch@zeha.at>
+ *
+ * SPDX-License-Identifier: LGPL-2.1-or-later
+ */
+
+#include "config.h"
+
+#include <stdio.h>
+
+#include "fu-mntre-device.h"
+#include "mntre_reset.h"
+
+struct _FuMntreDevice {
+	FuUsbDevice parent_instance;
+};
+
+G_DEFINE_TYPE(FuMntreDevice, fu_mntre_device, FU_TYPE_USB_DEVICE)
+
+static gboolean
+fu_mntre_device_get_firmware_version(FuMntreDevice *self,
+				     GError **error,
+				     struct mntre_reset_firmware_version *firmware_version)
+{
+	g_autoptr(GError) error_local = NULL;
+	gsize actual_len = 0;
+
+	guint16 reset_interface = 2; // TODO
+
+	FuUsbDeviceClaimFlags flags = 0;
+
+	if (!fu_usb_device_claim_interface(FU_USB_DEVICE(self), reset_interface, flags, error)) {
```

Comment:
this is normally done in _setup automatically -- can you try using `fu_usb_device_add_interface(FU_USB_DEVICE(self), 2)` in __init() and removing this?

Reply:
we need to discover the "2". have to see if that can be done in __init.

Reply:
Aha, I've added this in probe to find it via the class/subclass/protocol.